test/services/UpdateAddressResponseSpec.scala

```diff
@@ -46,27 +49,27 @@ class UpdateAddressResponseSpec extends BaseSpec with I18nSupport with MockitoSu
   def genericFunc(): Result =
     Ok
 
-  "UpdateAddressResponse.response" should {
+  "UpdateAddressResponse.response" must {
     "return the block result for UpdateAddressSuccessResponse" in {
-      val result = UpdateAddressSuccessResponse.response(genericFunc)
-      status(result) shouldBe OK
+      val result = Future(UpdateAddressSuccessResponse.response(genericErrors, genericFunc))
+      status(result) mustBe OK
     }
 
     "return BAD_REQUEST for UpdateAddressBadRequestResponse" in {
-      val result = UpdateAddressBadRequestResponse.response(genericFunc)
-      status(result) shouldBe BAD_REQUEST
+      val result = Future(UpdateAddressBadRequestResponse.response(genericErrors, genericFunc))
+      status(result) mustBe BAD_REQUEST
     }
 
     "return INTERNAL_SERVER_ERROR for UpdateAddressUnexpectedResponse" in {
       val updateAddressResponse = UpdateAddressUnexpectedResponse(HttpResponse(123))
-      val result = updateAddressResponse.response(genericFunc)
-      status(result) shouldBe INTERNAL_SERVER_ERROR
+      val result = Future(updateAddressResponse.response(genericErrors, genericFunc))
+      status(result) mustBe INTERNAL_SERVER_ERROR
     }
 
     "return INTERNAL_SERVER_ERROR for UpdateAddressErrorResponse" in {
       val updateAddressResponse = UpdateAddressErrorResponse(new RuntimeException("not used"))
-      val result = updateAddressResponse.response(genericFunc)
-      status(result) shouldBe INTERNAL_SERVER_ERROR
+      val result = Future(updateAddressResponse.response(genericErrors, genericFunc))
```

Comment:
Why has Future been wrapped around this?

Reply:
Done